bucket/neovim.json

```diff
@@ -0,0 +1,35 @@
+{
+    "homepage": "https://neovim.io/",
+    "license": "https://github.com/neovim/neovim/blob/master/LICENSE",
+    "version": "0.2.2",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/neovim/neovim/releases/download/v0.2.2/nvim-win32.zip",
+            "hash": "edbb17047f9a1bdc01acf965430ebbc61bd05a9d39668c9cc6961ea47f471c47"
+        },
+        "64bit": {
+            "url": "https://github.com/neovim/neovim/releases/download/v0.2.2/nvim-win64.zip",
+            "hash": "66bab53ec0a8aff854921b6925630026649f3a49bb7913d19a4a602ffc138c92"
+        }
+    },
+    "extract_dir": "Neovim",
+    "bin": [
+        "bin\\nvim.exe",
+        "bin\\nvim-qt.exe"
+    ],
+    "checkver": {
+        "github": "https://github.com/neovim/neovim"
```

Comment:
https://github.com/neovim/neovim/releases/latest redirects to the latest tag, maybe that's useful.

Reply:
The `github` utilizes the `/releases/latest` part 👍